pkg/controller/build/controller_test.go

```diff
@@ -232,12 +232,13 @@ func TestTimeoutFlows(t *testing.T) {
 	bldr := &nop.Builder{}
 
 	build := newBuild("test")
-	addBuffer, err := time.ParseDuration("10m")
+	buffer, err := time.ParseDuration("10m")
 	if err != nil {
 		t.Errorf("Error parsing duration")
 	}
 
-	build.Status.StartTime.Time = metav1.Now().Time.Add(-addBuffer)
+	// Set creation time to Now()-10min
+	build.Status.CreationTime.Time = metav1.Now().Time.Add(buffer)
```

Comment:
Previously this was `-addBuffer`, now it's `buffer`, can you explain why this is?

Reply:
It made more sense to call the variable as buffer. Ideally I want to subtract the buffer time to trigger timeout.

Reply:
Also just realized that line is not required for test setup.  Let me update.